### PR TITLE
Fixed package name and added additional packages

### PR DESCRIPTION
**Pull request by marcel-mathias-nolte**

### CODE REVIEW

File: meta/marcel-mathias-nolte/ken-burns-slideshow/de.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+de:
+  title: Zoom Dia-Show
+  description: >
+    Bietet eine Dia-Show mit Zoom--Effekt, welche im Voll-Bild-Modus angezeigt wird.
```
Comment (xchs) [inline meta/marcel-mathias-nolte/ken-burns-slideshow/de.yml:4]:
```suggestion
    Bietet eine Dia-Show mit Zoom-Effekt, welche im Vollbildmodus angezeigt wird.
```

File: meta/marcel-mathias-nolte/contao-filesmanager-fileusage/en.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+en:
+  title: Show file references
+  description: >
+    Displays an icon, which shows whether an file is used anywhere or not and shows details on where it is used if you click on it.
```
Comment (xchs) [inline meta/marcel-mathias-nolte/contao-filesmanager-fileusage/en.yml:4]:
```suggestion
    Displays an icon, which shows whether a file is used anywhere or not and shows details on where it is used if you click on it.
```

Comment (marcel-mathias-nolte) [reply meta/marcel-mathias-nolte/contao-filesmanager-fileusage/en.yml:4]: I tried this, but then Lint failed

**Comment by marcel-mathias-nolte** [comment]
How can 'Vollbildmodus' be added to the LINT whitelist?

**Comment by xchs** [comment]
> How can 'Vollbildmodus' be added to the LINT whitelist?

Just add it here: https://github.com/contao/package-metadata/blob/main/linter/allowlists/de.txt

**Review by xchs — APPROVED** [review]

**Pull request merged**